compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -1777,5 +1777,15 @@ object messages {
     val explanation =
       hl"""A class marked with the ${"final"} keyword cannot be extended"""
   }
-
+  case class EnumCaseDefinitionInNonEnumOwner(owner: Symbol)(implicit ctx: Context)
+    extends Message(EnumCaseDefinitionInNonEnumOwnerID) {
+      val kind = "Syntax"
+      val msg = em"case not allowed here, since owner `${owner}` is not an `${"enum"}` object"
```

Comment:
You shouldn't backtick `owner` and `enum`

Reply:
should be fixed